tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "noEmit": true,
     "baseUrl": ".",
     "allowSyntheticDefaultImports": true,
+    "types": ["node", "jest", "@testing-library/jest-dom"],
```

Comment:
These are more changes to fix some warnings from TypeScript and are unrelated to the pagination work.